usr/sbin/registercloudguest

```diff
@@ -70,13 +110,23 @@ def register_modules(extensions, products, registered=[], failed=[]):
         triplet = '/'.join((identifier, version, arch))
         if triplet in products and triplet not in registered:
             registered.append(triplet)
-            cmd = [
-                register_cmd,
-                '--url',
-                'https://%s' % registration_target.get_FQDN(),
-                '--product',
-                triplet
-            ]
+            if 'transactional' in register_cmd:
+                cmd = [
+                    register_cmd,
+                    sub_cmd,
```

Comment:
Not sure if it is a GH issue but I do not see where `sub_cmd` is set ?

Reply:
Good catch, thanks. Fixed. should have been `register` as in the other case.